<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Стек №1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Стек №2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Черга №1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Черга №2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>669</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="stack1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="stack2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="queue1">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="queue2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>471</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="b1">
       <property name="text">
        <string>Заповнити стек №1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b2">
       <property name="text">
        <string>Змінити дані в стеку №1 </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b3">
       <property name="text">
        <string>Заповнити стек №2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b4">
       <property name="text">
        <string>Змінити стек №1 за допомогою даних з стеку №2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b5">
       <property name="text">
        <string>Відсортувати стек №1 по спаданню</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b6">
       <property name="text">
        <string>Знайти елемент що задовільняє умові</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b7">
       <property name="text">
        <string>Копіюваити стек №1 в чергу №1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b8">
       <property name="text">
        <string>Відсортувати стек №1 та чергу №1 по зростанню</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b9">
       <property name="text">
        <string>Злити стек №1 та стек №2 в чергу №2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b10">
       <property name="text">
        <string>Вивести к-сть з черги №2 що задовільняють умову</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="b2_n">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>290</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b2_m">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>290</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b4_pos">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b4_n">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>350</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="b6_res">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="b10_res">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>540</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
